modules/auxiliary/gather/elasticsearch_enum.rb

```diff
@@ -0,0 +1,426 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+  include Msf::Module::Deprecated
+
+  moved_from 'auxiliary/scanner/elasticsearch/indices_enum'
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'ElasticSearch Enumeration Utility',
+        'Description' => %q{
+          This module enumerates ElasticSearch instances. It uses the REST API
```

Comment:
```suggestion
        'Name' => 'Elasticsearch Enumeration Utility',
        'Description' => %q{
          This module enumerates Elasticsearch instances. It uses the REST API
```

It seems to be spelled with a lowercase S. https://www.elastic.co/elasticsearch/

Reply:
I had that originally, but the previous module had the S, so I figured I was wrong....